src/linearmixedmodel.jl

```diff
@@ -499,6 +499,33 @@ Test whether the model `m` is singular if the parameter vector is `θ`.
 """
 issingular(m::LinearMixedModel, θ=m.θ) = any(isapprox.(lowerbd(m), θ))
 
+function StatsBase.leverage(m::LinearMixedModel{T}) where {T}
```

Comment:
This is definitely something that's nice to have, but what made you think of this here (=as part of this PR)?